src/main/java/io/kabanero/api/InstanceEndpoints.java

```diff
@@ -125,10 +130,61 @@ public Response isAdmin(@PathParam("instanceName") String instanceName) throws I
         return Response.ok(body).build();
     }
 
+    @GET
+    @Path("{instanceName}/admin")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getAdminList(@PathParam("instanceName") String instanceName) throws IOException, ApiException, GeneralSecurityException {
+        UserProfile userProfile = UserProfileManager.getUserProfile();
+        String token = userProfile.getAccessToken();
+        GitHubClient client = new GitHubClient();
+        client.setOAuth2Token(token);
+
+        Kabanero instance = KabaneroClient.getAnInstance(instanceName);
+        if (instance == null) {
+            return Response.status(404).entity(new ResponseMessage(instanceName + " not found")).build();
+        }
+
+        String instanceGithubOrg = instance.getSpec().getGithub().getOrganization();
+        List<String> instanceGithubTeams = instance.getSpec().getGithub().getTeams();
+
+        List<Object> instanceTeamMembers = new ArrayList<>();
+        List<User> allInstanceAdmins = new ArrayList<>();
+
+        TeamService teamService = new TeamService(client);
+
+        // Loop though all teams in the Kabanero CRD orginization and check to if the team exists in the list of Kabanero crd teams
+        for (Team orgTeam : teamService.getTeams(instanceGithubOrg)) {
+            for (String instanceAdminTeam : instanceGithubTeams) {
```

Comment:
instead of having another loop here I think there should be a `instanceGithubTeams.contains(orgTeam.getName())` or something. It will help clear up this section a little by removing a loop.